spec/fabricators/reviews_fabricator.rb

```diff
@@ -1,6 +1,6 @@
 Fabricator(:review) do 
-  video_id {Fabricate(:video).id}
-  user_id {Fabricate(:user).id}
+  video {Fabricate(:video)}
```

Comment:
I don't think you need to specify `{Fabricate(:video)}`, it should work with just `video`.